traitsui/tests/editors/test_csv_editor.py

```diff
@@ -100,8 +100,6 @@ def _qt_get_text_value(ui):
             expected = csv_list_editor._format_list_str([1.0, 3.14])
             self.assertEqual(value_str, expected)
 
-            press_ok_button(ui)
```

Comment:
For the record, this `press_ok_button` may be more accurately named "press_a_button". It does not check if the button it found is really the "OK" button. The intention of doing this in the test was to close the UI dialog, and that is now done in `create_ui`.

This PR does not intend to remove all `press_ok_button`, there are still a few lying around. One thing at a time...